code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -156,7 +156,7 @@ Doesn't work on other aliens/AI.*/
 
 /obj/effect/proc_holder/alien/acid/proc/corrode(atom/target,mob/living/carbon/user = usr)
 	if(target in oview(1,user))
-		if(target.acid_act(200, 1000))
+		if(target.acid_act(400, 1000))
```

Comment:
Turfs don't really care about acid power beyond whether it's greater than ~200 IIRC. The actual turf melting bit is just a counter on the acid component. I think xeno acid was already hitting the acid damage cap for objects as well, so this might not do anything.

Reply:
Seems faster to me.